Modules/Core/SpatialObjects/test/itkEllipseSpatialObjectTest.cxx

```diff
@@ -127,17 +127,19 @@ int itkEllipseSpatialObjectTest(int, char* [])
   }
   std::cout<<"[PASSED]"<<std::endl;
 
-  myEllipse->ComputeFamilyBoundingBox( EllipseType::MaximumDepth );
+  //NOTE: ORDER OF Update() and ComputeFamilyBoundingBox() is important.
   myEllipse->Update();
+  myEllipse->ComputeFamilyBoundingBox( EllipseType::MaximumDepth );
   const EllipseType::BoundingBoxType * boundingBox
     = myEllipse->GetFamilyBoundingBoxInWorldSpace();
   std::cout << "Bounds = " << boundingBox->GetBounds() << std::endl;
 
   std::cout << "Update(): ";
   for(unsigned int i=0;i<3;i++)
   {
-    if(   itk::Math::NotAlmostEquals(boundingBox->GetBounds()[2*i], 7 )
-       || itk::Math::NotAlmostEquals(boundingBox->GetBounds()[2*i+1], 16 )
+    const EllipseType::BoundingBoxType::BoundsArrayType bnds = boundingBox->GetBounds();
```

Comment:
not much was saved by abbreviating `bounds` to `bnds`

Reply:
It allowed me to see the variable in the debugger when I was trying to figure out what was going wrong.

Reply:
That is a reason I frequently introduce a variable too. But you could have named it `bounds` instead of `bnds`. You saved 2 characters, but made the name more cryptic.

Reply:
Ok.  I fixed the variable name to be bounds.